src/winetricks

```diff
@@ -4033,6 +4033,10 @@ winetricks_showmenu()
                         installed=TRUE
                         echo "${code}" >> "${WINETRICKS_WORKDIR}"/installed.txt
                     fi
+                    if [ "${#title}" -gt 100 ]; then
+                        title=$(printf "%s" "${title}" | head -c 90)
```

Comment:
What's the reasoning behind choosing two different numbers here?

Reply:
It's a hysteresis to not replace the last few characters or last word by three dots.

Reply:
Makes sense; could you please add a comment to that effect, for future reference?

Reply:
I'd also suggest changing 90 > 95, but I'll leave that to your discretion since you're actually using the GUI ;).

Reply:
Alright. PR updated.